assets/docker/Dockerfile.go.tmpl

```diff
@@ -3,7 +3,7 @@ FROM golang:{{ or .RuntimeVersion .DefaultRuntimeVersion }} as build
 WORKDIR /go/src/app
 COPY . .
 
-RUN go mod download
+RUN {{ .InstallCommand }}
```

Comment:
We don't need to move this to be templated, the task is only about the nodejs dockerfile